content/get-involved.ts

```diff
@@ -2,6 +2,13 @@ import {
   AirtableEmbed, BenefitItem, Button, FAQ, Header, MetaPageHead, PerfectFitCard,
 } from './content.types.ts';
 
+export interface Testimonies {
```

Comment:
this interface can be moved to `./content.types.ts` and imported into this file to align with the other custom interfaces like on `ln 2`. It may also be better to rename it to `TestimonyCard` or something like that, as it is the interface for an individual `Testimony`.

Reply:
Thanks for bringing that to my attention. I forgot to make it singular. Done.